### PR TITLE
Add compat data for ParentNode

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/ParentNode

### CODE REVIEW

File: api/ParentNode.json [hunk 1]
```diff
@@ -0,0 +1,678 @@
...
+            }
+          }
+        }
+      },
+      "append": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode/append",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/ParentNode.json]: Please change this to `"54"`. https://www.chromestatus.com/features/5723067795898368

File: api/ParentNode.json [hunk 2]
```diff
@@ -0,0 +1,678 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "prepend": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode/prepend",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/ParentNode.json]: 54

File: api/ParentNode.json [hunk 3]
```diff
@@ -0,0 +1,678 @@
...
+              "version_added": "49"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "39"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/ParentNode.json]: 39

File: api/ParentNode.json [hunk 4]
```diff
@@ -0,0 +1,678 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "querySelectorAll": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode/querySelectorAll",
+          "support": {
+            "webview_android": {
+              "version_added": "2.1"
```
Comment (jpmedley) [inline api/ParentNode.json]: Please change this to `true`. Webview follows Chrome numbering, not Android numbering and this is older than I can get a specific version onl

File: api/ParentNode.json [hunk 5]
```diff
@@ -0,0 +1,678 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/ParentNode.json]: This should be `true`.

File: api/ParentNode.json (hunk 1 above)
Comment (maboa) [reply api/ParentNode.json]: 👍

File: api/ParentNode.json (hunk 2 above)
Comment (maboa) [reply api/ParentNode.json]: 👍

File: api/ParentNode.json (hunk 3 above)
Comment (maboa) [reply api/ParentNode.json]: 👍

File: api/ParentNode.json (hunk 4 above)
Comment (maboa) [reply api/ParentNode.json]: 👍

File: api/ParentNode.json (hunk 5 above)
Comment (maboa) [reply api/ParentNode.json]: 👍

**Comment by maboa** [comment]
Changes should be made now :)

**Pull request merged**